tests/functional/include/tests/ft_utils.h

```diff
@@ -0,0 +1,21 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "rtos/ThisThread.h"
+
+#include "boost/ut.hpp"
+
+using namespace std::chrono;
+using namespace boost::ut;
+
+void busy_sleep(uint64_t milliseconds)
+{
+	for (uint64_t now = 0; now < milliseconds; ++now) {
+		rtos::ThisThread::sleep_for(1ms);
```

Comment:
do you need `1ms`, can't you make it 10 or 20? 

when you call `busy_sleep(100)` it means 100 dots will be printed, which is a lot. Even worst when you call is with `busy_sleep(2000)`

I think if `100ms` is okay, it would be better.

Reply:
It does not bother but so the parameter will be a multiplier

Reply:
hmm, no it will not.

```cpp
void busy_sleep(std::chrono::milliseconds duration)
{
	static constexpr sleep_delay = 500ms;
	for (uint64_t now = 0; now < duration/sleep_delay; ++now) {
		rtos::ThisThread::sleep_for(sleep_delay);
	//...
```